tests/ui/uninlined_format_args.rs

```diff
@@ -147,11 +145,16 @@ fn tester(fn_arg: i32) {
     );
     println!(no_param_str!(), local_i32);
 
-    // FIXME: bugs!
-    // println!(pass_through!("foo={local_i32}"), local_i32 = local_i32);
-    // println!(pass_through!("foo={}"), local_i32);
-    // println!(indoc!("foo={}"), local_i32);
-    // printdoc!("foo={}", local_i32);
+    println!(
+        "{}",
+        // comment with a comma , in it
+        val,
+    );
+    println!("{}", /* comment with a comma , in it */ val);
+    println!("{named}", named = val);
```

Comment:
why do you need this test case?  It's being covered above too.

Reply:
Yeah I'll remove that one, was just leftover from when I was testing the implementation in another file